week1.html

```diff
@@ -28,17 +28,477 @@
             <div id="cart"><b>Cart</b></div>
         </a>
     </header>
+    <script>
```

Comment:
Instead of writing JS code inside the script block, you can move it to a separate JS file and link it here. That will make the code more maintainable

Reply:
done